packages/aws-cdk-lib/aws-route53/README.md

```diff
@@ -220,6 +220,37 @@ Constructs are available for A, AAAA, CAA, CNAME, MX, NS, SRV and TXT records.
 Use the `CaaAmazonRecord` construct to easily restrict certificate authorities
 allowed to issue certificates for a domain to Amazon only.
 
+### Health Checks
+
+You can associate health checks with records:
```

Comment:
A brief description of what health checks are and what their purpose is would be ideal here, even if failovers aren't supported yet